src/useBaseFetch.js

```diff
@@ -1,15 +1,15 @@
 import { useState } from 'react';
 import axios from 'axios';
 
-export default config => {
+export default url => {
```

Comment:
With this change, how would a different method other than `GET` be passed in?

Reply:
Yeah, good catch. This should actually be `axios`, not `axios.get`.